cypress/fixtures/fireboltMocks/clearEventData.json

```diff
@@ -42,5 +42,9 @@
     "SECONDSCREEN_ONCLOSEREQUEST_EVENT":{
         "event": "secondscreen.onCloseRequest",
         "firstParty": false
+    },
+    "ADVERTISING_ONSKIPRESTRICTIONCHANGED_EVENT":{
+        "event": "manage_advertising.onSkipRestrictionChanged",
+        "firstParty": false
     }
```

Comment:
why is firstParty false ? this looks to be a manage event . 

is this for clearing event listener on FCA , but for a manage event ?

Reply:
if this is applicable only for xumo config module, cant we place this in xumo module and not directly in fcs >?

Reply:
move this to config module and firstParty is false because making a call to 3rd party app